client/src/stories/docs/browser-support-matrix.stories.mdx

```diff
@@ -0,0 +1,39 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+
+<Meta title="Docs/Browser Support Matrix" />
+
+# Our browser support commitment to you
+
+MDN Web Docs is a resource for everyone and should therefore be accessible to the widest audience possible. With that said, we cannot support every version of every browser and have therefore adopted the following supprt matrix.
+
+Our commitment is to always offer the best possible user experience to all our readers. Where the work and code required to offer the same experience to users of older technologies will either be unmaintainable or cause a degraded experience due to slow performance, we will opt to offer a downgraded experience.
+
+Our main focus is to bring you the best web developer learning materials and documentation on the web, and no matter what browser or device you use to access MDN Web Docs, our content _will_ always be accessible.
+
+## Browser Support Matrix Breakdown
+
+## A Grade
+
+- Firefox (Latest two versions on the release channel)(Mobile and Desktop)
+- Chrome (Latest two versions on the release channel)(Mobile and Desktop)
+- Safari (Latest two versions on the release channel)(Mobile and Desktop)
+- Edge (Latest two versions on the release channel)(Mobile and Desktop)
+
+## B Grade
+
+Internet Explorer 11 and versions of modern browsers older than the above.
+
+This means that:
+
+- All content must be accessible. (for us this might mean not hydrating on the client and serve little to no JS and only essential CSS(reader mode))
```

Comment:
```suggestion
- All content must be accessible. For example, this might mean not hydrating on the client and serving little to no JS and only essential CSS (reader mode).
```